.env.example

```diff
@@ -1,12 +1,16 @@
+# Database
 DATABASE_URL=
-# JWT_SECRET_KEY=
 
+# Auth
 AUTH0_AUDIENCE=
 AUTH0_DOMAIN=
 
+# Cloud
 STORAGE_TYPE=
-
 AWS_DEFAULT_REGION=
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
```

Comment:
Is there any risk storing such variable on the front-end?

Reply:
But here is our backend application.